resources/docs/accounts.md

```diff
@@ -0,0 +1,90 @@
+
+To set it up:
+
+1. Go to **Accounts → Add New Account**. Select **Nextcloud News**.
+2. Enter the required information:
+    - URL: `https://nextcloud.server.example`. Do not enter any API or WebDAV information. RSS Guard will look for it automatically.
+    - Download only unread messages
+    - Force execution of server-side update when updating feeds from RSS Guard forces Nextcloud to refresh all the feeds stored. Depending on how many system resources are available to Nextcloud this _will_ cause you issues.
+    - Limit number of downloaded messages per feed. Limiting this is useful if you're on a mobile or rate-limited connection.
+    - Username: As it would be to log in to your Nextcloud instance.
+    - Password
+
+The minimum supported version of Nextcloud News is 6.0.5; any version of Nextcloud released in the last year should be compatible.
+
+## Standard online feeds (RSS/RDF/ATOM)
+
+This account type lets RSS Guard function as a stand-alone RSS client. It saves feeds and their state to the database and does not sync it with any external servers or applications. This _can_ be useful if you wish to sync the entire RSS Guard database if you do not plan on using any of the available integrations. Be aware the database can grow to huge sizes and you will see increased network traffic as your syncing application will have to keep up with the manipulation of feeds within RSS Guard itself.
+
+1. Go to **Accounts → Add New Account**. Select Standard online feeds (RSS/RDF/ATOM)
+2. You will be given the option to load a default set of feeds. If you select **Yes** the following folders and feeds will be created:
+    - Fun & Life: [Interesting Thing of the Day](http://feeds.feedburner.com/InterestingThingOfTheDay), [Mashable](http://feeds.feedburner.com/Mashable?format=xml), [SimplyRecipes.com](http://feeds.feedburner.com/elise/simplyrecipes)
+    - News: [BBC International News](http://feeds.bbci.co.uk/news/rss.xml?edition=int), [CNN World News](http://rss.cnn.com/rss/edition_world.rss)
+    - RSS Guard: [Recent Commits to rssguard:master](https://github.com/martinrotter/rssguard/commits/master.atom)
+    - Technology: [Linux Today](http://feeds.feedburner.com/linuxtoday/linux?format=xml), [TechCrunch](http://feeds.feedburner.com/techcrunch)
+
+## Gmail API
+
+This integration lets you download and read email messages stored in Gmail by making use of its [JSON API](https://developers.google.com/gmail/api). It also lets you send email messages. To make use of it follow the steps stated in [Gmail API documentation](https://developers.google.com/gmail/api/quickstart/js) or by configuring it directly in the [Google APIs Console](https://console.developers.google.com/apis/credentials):
+
+1. Enable Gmail API if it isn't already.
+2. Create a **Client ID** in the Google API console.
```

Comment:
Note that "client ID" and "API key" are technically the same things as "Application ID" and "Application key" in Inoreader account setup - it's just that guys from Inoreader call them differently than guys from Google, which sucks.

https://www.oauth.com/oauth2-servers/client-registration/client-id-secret/